lib/views/walks/walks_view.dart

```diff
@@ -46,13 +46,14 @@ class _WalksViewState extends State<WalksView> with WidgetsBindingObserver {
   LatLng? _homePosition;
   _ViewType _viewType = _ViewType.list;
   WalkFilter? _filter;
-  Future? _newsRunning;
 
   @override
   void initState() {
+    WidgetsBinding.instance!.addObserver(this);
     initializeDateFormatting("fr_BE");
     _retrieveData();
-    WidgetsBinding.instance!.addObserver(this);
+    _news();
+    _firstLaunch();
```

Comment:
I think it is better to let `_firstLaunch()` call in `_retrieveWalksHelper() ` like it was before, as it was working perfectly fine there

Reply:
Np, I'll put it back